.markdownlintrc

```diff
@@ -4,7 +4,9 @@
     "style": "atx"
   },
   "MD013": false,
-  "MD024": false,
+  "MD024": {
+    "allow_different_nesting": true,
```

Comment:
I assume the trailing ',' is wrong in this line 8 after 'true'.

-> "MD024": { "allow_different_nesting": true};

Reply:
Oh yes it is, thanks for the hint!